tests/MongoDB.Driver.Core.Tests/ChangeStreamDocumentTests.cs

```diff
@@ -72,6 +72,18 @@ public void BackingDocument_should_return_expected_result()
             result.Should().BeSameAs(backingDocument);
         }
 
+        [Fact]
+        public void BackingDocument_should_allow_duplicate_elements_in_full_document()
```

Comment:
>The normal cases
>Any unusual cases

as for me, this is a quite blurry rule. I'm not sure that we will be always on the same page in determining whether the current test is about a normal or unusual case. It can give us unnecessary debates about it. I would vote to have more clear rules.

Reply:
Maybe we can live with this..